omega/linear_regression/LinearRegressionClassifier.py

```diff
@@ -0,0 +1,124 @@
+import torch as ch
+import logging
+from torch.autograd import Variable
+
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class LinearRegressionClassifier(object):
+    """Objects of this class is a LinearRegressionClassifier.
+
+   Examples
+   --------
+   >>> import omega as omg
+   >>> import torch as ch
+   >>> x = ch.Tensor([[24], [50], [15], [55], [14], [12], [18], [62]])
+   >>> y = ch.Tensor([[21.54945196], [47.46446305], [17.21865634], [52.789],
+                    [16.1234], [12.789], [19.5649], [60.5793278]])
+   >>> lr = omg.linear_regression.LinearRegressionClassifier(x,y)
+   >>> lr.fit('Adam')
+   >>> lr.predict(x)
+
+   """
+
+    def __init__(self, x, y):
+        """instanciate an object with two parameters.
+
+        Parameters
+        ----------
+        x : a float tensor having area of house in square meter.
+        y : a float tensor having price on the basis of area of house.
+
+        """
+        self.x = x
+        self.y = y
+
+    def fit(self, option):
+        """ Finds suitable value of parameters which best fit the given output values.
+
+        Parameters
+        ----------
+        option : We have implemented in two ways. The first approach is the Analytical approach
+                 which computes the value of slope and intercept by finding the covariance
+                 and variance of the (x, y) pair. The second approach uses two optimizers namely
+                 stochastic gradient descent (SGD) and Adam. Therefore, the options are 'Analytical'
+                 'SGD' and 'Adam'.
+
+        """
+
+        self.option = option
+        if option == 'Analytical':
+            self.cov_x_y = []
+            self.var_x = []
+            mean_x = ch.mean(self.x)
+            mean_y = ch.mean(self.y)
+            for i in range(len(self.x)):
+                self.cov_x_y.append((self.x[i] - mean_x) * (self.y[i] - mean_y))
+                self.var_x.append((self.x[i] - mean_x) ** 2)
+            self.beta = sum(self.cov_x_y) / sum(self.var_x)
+            self.alpha = mean_y - self.beta * mean_x
+            return
+        ch.manual_seed(2)
```

Comment:
Please dont hard code the `seed`. Check out `nn`'s implementation for the use of seed.